gips/0080-gip-direct-indexing-payments.md

```diff
@@ -0,0 +1,147 @@
+- The Subgraph Service releases payments conditional on Indexers posting POIs - this ensures that Indexers cannot collect payment without doing the work (and if they present an invalid POI, they can be slashed). The voucher release contract can ensure that only up to the maximum amount is released, but the Subgraph Service can release the exact amount based on the agreed pricing (e.g. price per gas times the reported indexing gas used).
+
+**This voucher release mechanism is generic and can be used for other data services:** it validates that the caller is the approved data service contract, that the token amount is within the maximum value specified by the voucher per period, and that the right amount of time has passed. The data service specific aspects e.g. POI validity, gas/pricing, etc. are defined at the data service contract level. Moreover, this contract can be immutable, so payers are protected even if the data service contract is upgraded.
+
+It is important to note that we cannot expect Indexers to take on risk, e.g. by agreeing to a fixed price for a specific subgraph (instead of per unit of work/gas). The cost of indexing each subgraph is unknown until the subgraph has been indexed, so if the price was fixed a rational Indexer would simply stop indexing when the subgraph becomes unprofitable, forcing the payer to start over. Tying payments as much as possible to the verifiable cost of performing the work is the best way to prevent Denial of Service scenarios, just like with gas in Ethereum.
+
+### Terms and Definitions
+
+**Curation**: Curation requires Consumers to stake a certain amount of GRT per subgraph to signal Indexers to provide indexing services.  However, because the strength of the signal depended upon the strength of other signals using GRT, which also had variability, this became a hard mechanism for developers with many subgraphs.
+
+**Indexers:** Indexers are node operators in The Graph Network that stake Graph Tokens (GRT) in order to provide indexing and query processing services. Indexers earn query fees and indexing rewards for their services.
+
+**Indexing Network Payments:**  this represents the payments to Indexers upon verifying successful indexing.  Typically will occur between a Gateway and Indexers selected by their IISA.
+
+**End Consumers:** The core users of data services via Products built onto the Protocol are referred to as Consumers. Specifically, the consumer is the one that receives a benefit (and thus is willing to pay) for data services. In some cases, this may be the developer. In other cases, a consumer may want to leverage the subgraph of another developer. In general, the consumer is the party that interacts with Gateways/Indexers for the indexing services.
+
+**Consumer**: This is referring to one who interacts directly with the Protocol for indexing services.  Often a **Gateway** (see below).
```

Comment:
I think this can cause a bit of confusion with the "End Consumer" above. Maybe we should distinguish "End Consumers" or "Consumers at the Product level" from "Consumers at the Protocol level, usually Gateways"?